cluster/images/etcd-version-monitor/etcd-version-monitor.yaml

```diff
@@ -7,7 +7,6 @@ spec:
   hostNetwork: true
   containers:
   - name: etcd-version-monitor
-    image: k8s.gcr.io/etcd-version-monitor:0.1.3
+    image: staging-k8s.gcr.io/etcd-version-monitor:0.1.3
```

Comment:
hmm? we should be _pushing_ to staging, but not serving from it ??

Reply:
Should we?  I'm confused.

Reply:
AFAIK, yes. k8s.gcr.io is the prod registry, the staging registry is where built images are pushed before promotion to prod.

Reply:
xref: https://github.com/kubernetes-sigs/k8s-container-image-promoter

we'll be moving all of this to CNCF infra with the open sourced image promoter